.eslintrc.yml

```diff
@@ -8,7 +8,7 @@ extends:
 plugins:
   - import
 parserOptions:
-  ecmaVersion: "latest"
+  ecmaVersion: 14
```

Comment:
Keep the "latest" in there. 

TypeScript would tell us if the given functionality is not supported

Reply:
We were getting some false positives from the linter, this was a bypass. I can no longer replicate the problem though, changing it back.